src/assets/main.css

```diff
@@ -1,35 +1,15 @@
-@import './base.css';
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
 
-#app {
-  max-width: 1280px;
-  margin: 0 auto;
-  padding: 2rem;
-
-  font-weight: normal;
-}
-
-a,
-.green {
-  text-decoration: none;
-  color: hsla(160, 100%, 37%, 1);
-  transition: 0.4s;
+.btn {
```

Comment:
this file should not contain any css
if you need apply some styles for your components then create component that will contain`theme` property and make necessary styling inside component. this will prevent mess of styles and you will have full control of theme of your application